sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -174,6 +221,35 @@ public partial class ImportRedisEnterpriseDatabaseContent
         public ImportRedisEnterpriseDatabaseContent(System.Collections.Generic.IEnumerable<System.Uri> sasUris) { }
         public System.Collections.Generic.IList<System.Uri> SasUris { get { throw null; } }
     }
+    public partial class LocationInfo
```

Comment:
This is a quite general name. Please specify.
We could use `RedisEnterpriseLocationInfo` if nothing better comes to mind

Reply:
Updated